client/src/templates/Challenges/classic/desktop-layout.tsx

```diff
@@ -84,15 +89,23 @@ const DesktopLayout = (props: DesktopLayoutProps): JSX.Element => {
   const displayPreview = projectBasedChallenge
     ? showPreview && hasPreview
     : hasPreview;
+  const displayNotes = projectBasedChallenge ? showNotes && hasNotes : false;
   const displayConsole = projectBasedChallenge ? showConsole : true;
-  const { codePane, editorPane, instructionPane, previewPane, testsPane } =
-    layoutState;
+  const {
+    codePane,
+    editorPane,
+    instructionPane,
+    notesPane,
+    previewPane,
+    testsPane
+  } = layoutState;
 
   return (
     <div className='desktop-layout'>
-      {projectBasedChallenge && (
+      {hasEditableBoundaries && (
```

Comment:
Oliver might disagree with this change back.

Reply:
Probably, but it's possible we can simplify things and drop one of the two.

Sorry @moT01 this slipped under my radar. I'll give it a proper review on Monday.

Reply:
My thinking here is that if we don't use `showUpcomingChanges` as a test here, we will catch any potential problems sooner rather than later. I'll just change it back for now.

Reply:
No, you're probably right. Why don't we refactor out the `showUpcomingChanges` from the likes of `projectBasedChallenge` and see which ones we can already get rid of?  After this PR lands, though.